samples/highcharts/demo/column-basic/demo.js

```diff
@@ -3,67 +3,45 @@ Highcharts.chart('container', {
         type: 'column'
     },
     title: {
-        text: 'Monthly Average Rainfall'
+        text: 'Corn vs wheat estimated production for 2020 (1000 MT)'
     },
     subtitle: {
-        text: 'Source: WorldClimate.com'
+        text:
+            'Source: <a target="_blank" ' +
+            'href="https://www.indexmundi.com/agriculture/?commodity=corn">indexmundi</a>'
     },
     xAxis: {
-        categories: [
-            'Jan',
-            'Feb',
-            'Mar',
-            'Apr',
-            'May',
-            'Jun',
-            'Jul',
-            'Aug',
-            'Sep',
-            'Oct',
-            'Nov',
-            'Dec'
-        ],
-        crosshair: true
+        categories: ['USA', 'China', 'Brazil', 'EU', 'India', 'Russia'],
+        crosshair: true,
+        accessibility: {
+            description: 'Countries'
+        }
     },
     yAxis: {
         min: 0,
         title: {
-            text: 'Rainfall (mm)'
+            text: 'Metric tons (MT)'
         }
     },
     tooltip: {
-        headerFormat: '<span style="font-size:10px">{point.key}</span><table>',
-        pointFormat: '<tr><td style="color:{series.color};padding:0">{series.name}: </td>' +
-            '<td style="padding:0"><b>{point.y:.1f} mm</b></td></tr>',
-        footerFormat: '</table>',
-        shared: true,
-        useHTML: true
+        valueSuffix: ' (1000 MT)',
+        stickOnContact: true,
+        backgroundColor: 'rgba(255, 255, 255, 0.93)'
```

Comment:
Better leave out the background color I think, otherwise this doesn't work well with themes (Dark Unica in particular).